ci_build/steps_template.yml

```diff
@@ -0,0 +1,57 @@
+parameters:
+  python_version: '3.5'
+  tensorflow_version: ''
+  opset_version: ''
+  onnx_version: ''
+  onnxruntime_version: ''
+  test_case: '' # can be 'unit_tests', 'pretrained_models'
```

Comment:
onnxruntime_version is not generic, we may enable caffe2 some day.
test_case may not be exclusive, use a variable might be better.

Reply:
personally, i prefer to use variable for non generic purpose.
we may need to add new options in the future, and u don't need to update template parameters in ALL pipeline files.

Reply:
done